ktor-server/ktor-server-core/jvm/src/io/ktor/sessions/SessionTrackerById.kt

```diff
@@ -9,6 +9,8 @@ import io.ktor.util.*
 import io.ktor.utils.io.*
 import kotlin.reflect.*
 
+val SessionIdKey = AttributeKey<String>("SessionId")
```

Comment:
In ktor, we use so-called "explicit mode" so all visibility modifiers are required. So "public" is required, otherwise, a compiler error is reported

```suggestion
public val SessionIdKey = AttributeKey<String>("SessionId")
```

Reply:
Also explicitly specifying the type was required.